exchange_calendars/exchange_calendar_xbom.py

```diff
@@ -437,6 +437,7 @@
         "2024-10-02",
         "2024-11-01",
         "2024-11-15",
+        "2024-11-20", # Maharashtra Assembly Elections
```

Comment:
Build is failing due to lint issue:
```
./exchange_calendars/exchange_calendar_xbom.py:440:22: E261 at least two spaces before inline comment
```

Two spaces are required, rather than one, before the comment.
```suggestion
        "2024-11-20",  # Maharashtra Assembly Elections
```